book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Great job adding rentals in the constructor, here we also need a method (add_rental) to add rentals, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
`Rental.new(...)`